utils/soca/gdas_postprocincr.h

```diff
@@ -92,42 +92,49 @@ class PostProcIncr {
   }
 
   // -----------------------------------------------------------------------------
-  // Append layer thicknesses to increment
-  // TODO(guillaume): There's got to be a better way to append a variable.
-  soca::Increment appendLayer(const soca::Increment& socaIncr) {
+  // Append variable to increment
+  soca::Increment appendVar(const soca::Increment& socaIncr, const oops::Variables varToAppend) {
```

Comment:
Not for this PR: I agree, there's gotta be a better way to append a variable. Is this meant to add new zero variables? Looking through soca code, perhaps `soca::Increment::updateFields` could be enough? It looks like it's creating new variables if the argument contains new variables, and I think from the code that the fields for new variables would be set to zero.

Reply:
@shlyaeva , It was meant to append the layer thickness state to the mom6 iau increment at first, but I now need to add u,v if it's not in the soca incr. I'll have a look at `updateFields`

Reply:
no worries about looking into it now. I am also happy to look into this in the next few days/weeks, let me know.

Reply:
We'll have to revisit this at some point for sure. The LETKF increment isn't used currently, it needs to be combined with the re-centering, which is the only thing that we currently apply to the ensemble members.